cmd/subcommands/staking.go

```diff
@@ -91,11 +91,25 @@ func createStakingTransaction(nonce uint64, f staking.StakeMsgFulfiller) (*staki
 	if err != nil {
 		return nil, err
 	}
-	gasLimit, err := core.IntrinsicGas(data, false, true)
-	if err != nil {
-		return nil, err
+	var gLimit uint64
+	if gasLimit == "" {
+		fmt.Println("Using IntrinsicGas")
+		gLimit, err = core.IntrinsicGas(data, false, true, true)
```

Comment:
We are not always doing a create validator staking transaction. Use the directive from `staking.StakeMsgFulfiller` to check if it's a create validator staking txn